### PR TITLE
DELETE : division between two Shares

### PR DESCRIPTION
**Pull request by sakamoto-souta**
# Summary
DELETE : division between two Shares
# Purpose
to delete division which is unused
# Contents
delete division and the parts which depend on division
# Testing Methods Performed
make test t=./computation_container p=medium

### CODE REVIEW

**Review by KotaTakahashi9320 — CHANGES_REQUESTED** [review]
Thank you for your contribution!
Please check comments.

**Review by watason — APPROVED** [review]
good job!

File: packages/server/computation_container/share/share.hpp [hunk 1]
```diff
@@ -126,11 +131,13 @@ class Share : boost::operators<Share<SV>>,
 
         return *this;
     }
+    /*
     Share &operator/=(const Share &obj)
     {
         *this *= getInv(obj);
         return *this;
     }
+    */
```
[the remark this answers is not in the document]
Comment (sakamoto-souta) [reply packages/server/computation_container/share/share.hpp]: [f9c5143](https://github.com/acompany-develop/QuickMPC/pull/157/commits/f9c5143aef5c56d983f7df56296187554d01111b)

File: packages/server/computation_container/optimizer/BUILD
[the remark this answers is not in the document]
Comment (sakamoto-souta) [reply packages/server/computation_container/optimizer/BUILD:1]: [be87cdb](https://github.com/acompany-develop/QuickMPC/pull/157/commits/be87cdbbffaa38430f3cc13c963640874855f29a)

**Review by KotaTakahashi9320 — APPROVED** [review]
LGTM

**Pull request merged**